Pluses Server/src/ru/shemplo/pluses/logic/HelpHandler.java

```diff
@@ -0,0 +1,46 @@
+package ru.shemplo.pluses.logic;
+
+import static ru.shemplo.pluses.network.message.AppMessage.MessageDirection.*;
+import static ru.shemplo.pluses.network.message.ControlMessage.ControlType.*;
+import static ru.shemplo.pluses.Run.*;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+
+import ru.shemplo.pluses.network.message.ControlMessage;
+import ru.shemplo.pluses.network.message.Message;
+import ru.shemplo.pluses.network.pool.AppConnection;
+
+
+public class HelpHandler {
+	static private Long lastUpdated = 0L;
+	static private String commandsHelp;
+	
+	public static void runHelp(AppConnection connection) throws IOException {
+		tryUpdateHelpContent();
+		Message help = new ControlMessage (STC, INFO, 0, commandsHelp);
+        connection.sendMessage (help);
+	}
+	
+	private static void tryUpdateHelpContent() throws IOException {
+		File hf = new File(HELP_NAME);
+		Long modified = hf.lastModified();
+		if (lastUpdated >= modified) return;
+		
+		try(BufferedReader br = new BufferedReader(new FileReader(hf))) {
```

Comment:
unspecified file encoding (FileReader doesn't set encoding, by default it's "null")
```java
try (
    BufferedReader br = Files.newBufferedReader (path, StandardCharsets.UTF_8);
) {
    ...
}
```